firepit/config.template.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  /*
+    Headless mode forces the firepit builds to exactly imitate firebase-tools,
+    so the resulting binary "firebase" is a drop in replacement for the script
+    installed via npm. This is the behavior for CI / Cloud Shell / Docker etc.
+
+    When headless mode is disabled, the "double click" experience is enabled
+    which allows the binary to spawn a terminal on Windows and Mac. The is the
+    behavior for desktop users.
+  */
+  headless: headless_value,
```

Comment:
What is `headless_value` here?  And why is it not quoted but `"firebase_tools_package_value"` is.